<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainClass</class>
 <widget class="QMainWindow" name="AppMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AppMain</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mainLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aLoadPath"/>
    <addaction name="aSavePath"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aLoadPath"/>
   <addaction name="aSavePath"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="aLoadPath">
   <property name="icon">
    <iconset resource="AppMain.qrc">
     <normaloff>:/AppMain/Resources/Icons/openfile.ico</normaloff>:/AppMain/Resources/Icons/openfile.ico</iconset>
   </property>
   <property name="text">
    <string>Load Path</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="aSavePath">
   <property name="icon">
    <iconset resource="AppMain.qrc">
     <normaloff>:/AppMain/Resources/Icons/savefile.ico</normaloff>:/AppMain/Resources/Icons/savefile.ico</iconset>
   </property>
   <property name="text">
    <string>Save Path</string>
   </property>
  </action>
  <action name="aWorld">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>World</string>
   </property>
  </action>
  <action name="aTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="aTrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="aLinear">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="aCardinal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cardinal</string>
   </property>
  </action>
  <action name="aCubic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cubic</string>
   </property>
  </action>
  <action name="aLine">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="aTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </action>
  <action name="aRoad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Road</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppMain.qrc"/>
 </resources>
 <connections/>
</ui>
